test/conformance/network_policy_v2_test.go

```diff
@@ -50,16 +54,50 @@ func TestNetworkPolicyV2Conformance(t *testing.T) {
 	t.Log("Starting the network policy V2 conformance test suite")
 	// Depending on the tests some of them take longer and end up timing
 	// out on context, let's bump the GetTimeout to 300 seconds here.
-	cSuite := suite.New(suite.Options{
-		Client:                     client,
-		ClientSet:                  clientset,
-		KubeConfig:                 *cfg,
-		Debug:                      showDebug,
-		CleanupBaseResources:       shouldCleanup,
-		EnableAllSupportedFeatures: enableAllSupportedFeatures,
-		BaseManifests:              conformanceTestsBaseManifests,
-		TimeoutConfig:              netpolv1config.TimeoutConfig{GetTimeout: 300 * time.Second},
-	})
+	profiles := sets.Set[suite.ConformanceProfileName]{}
+	profiles.Insert(suite.ConformanceProfileName(suite.SupportAdminNetworkPolicy))
+	profiles.Insert(suite.ConformanceProfileName(suite.SupportBaselineAdminNetworkPolicy))
+	cSuite, err := suite.NewConformanceProfileTestSuite(
+		suite.ConformanceProfileOptions{
+			Options: suite.Options{
+				Client:                     client,
+				ClientSet:                  clientset,
+				KubeConfig:                 *cfg,
+				Debug:                      showDebug,
+				CleanupBaseResources:       shouldCleanup,
+				EnableAllSupportedFeatures: enableAllSupportedFeatures,
+				BaseManifests:              conformanceTestsBaseManifests,
+				TimeoutConfig:              netpolv1config.TimeoutConfig{GetTimeout: 300 * time.Second},
+			},
+			Implementation: confv1a1.Implementation{
```

Comment:
For anyone reviewing: https://github.com/kubernetes-sigs/network-policy-api/blob/f6c1cf24c0488f12c21920d2d9286fe0a380dd76/conformance/apis/v1alpha1/conformancereport.go#L75